libs/tracing/unix/Tracing.ml

```diff
@@ -87,8 +88,14 @@ let with_setup f =
      to ensure the trace_id is the same for all spans, but we decided that
      having the top level time is a good default. *)
   let url =
-    match Sys.getenv_opt endpoint_env_var with
-    | Some url -> url
+    match traces_endpoint with
+    | Some url -> (
+        match url with
+        | "semgrep-dev" -> default_dev_endpoint
+        | "semgrep-local" -> default_local_endpoint
+        | "dev" -> default_dev_endpoint
+        | "local" -> default_local_endpoint
```

Comment:
Thought about whether this makes sense; ultimately I think having only one alias for each option is better to avoid confusion. I'd be down for semgrep-dev and local since local is more universal.

Reply:
Also, I realized that for completeness + for the benchmarks it might be better to also have `semgrep-prod` explicitly